### ISSUE TITLE
Packing takes a long time

### ISSUE THREAD

**Issue by lhDream** [fvarrui/JavaPackager#365]
It took a very long time at this `Trying to resolve license from POM ...` position .

![image](https://github.com/fvarrui/JavaPackager/assets/41568266/57765ebe-ea3f-4529-9210-458dd66beab8)
![image](https://github.com/fvarrui/JavaPackager/assets/41568266/512770af-c922-4438-ab74-9075f4f1e532)

Sometimes the following exception message will be thrown, but it can be packaged normally .
```shell
[ERROR]             Cannot download license from https://www.apache.org/licenses/LICENSE-2.0
```

**Comment by fvarrui** [comment]
I've seen your PR and just merged into `devel` ... do you need a release with these changes?

**Comment by lhDream** [comment]
Don't worry, I can use local.

**Comment by fvarrui** [comment]
I just released 1.7.6.
Can we close this issue?

**Comment by lhDream** [comment]
sure
